packages/core/admin/admin/src/translations/en.json

```diff
@@ -626,6 +626,7 @@
   "components.Blocks.modifiers.underline": "Underline",
   "components.Blocks.modifiers.strikethrough": "Strikethrough",
   "components.Blocks.modifiers.code": "Code",
+  "components.Blocks.blocks.label": "Select a block",
```

Comment:
Actually this looks like "label" is a type of block. could you name it something like `components.Blocks.selectBlock`?